### ISSUE TITLE
Please have it in English please.

### ISSUE THREAD

**Issue by braysonjohn148** [aliyoge/flutter_file_preview#6]
Users are getting error messages in a language they cant read. Please

**Comment by evanswanyoike** [comment]
Unrelated but how did you add this package to your app. Can you share your pubspec.yaml facing errors when trying to build.

**Comment by braysonjohn148** [comment]
Yes, i know somebody else will have this issue please change the name to flutter_file_preview not like how the guy wrote there

from `file_preview` to `flutter_file_preview`

**Comment by evanswanyoike** [comment]
Managed to get it working with some plugin but now all opened files show "file not found" in chinese after opening any file. Did you also face that issue

**Comment by braysonjohn148** [comment]
You have to rebuild the app agin.

**Comment by evanswanyoike** [comment]
Many builds from scratch even after uninstalling the app from my phone

**Comment by evanswanyoike** [comment]
> You have to rebuild the app agin.

Can you share a screenshot of the working app

**Comment by braysonjohn148** [comment]
I am building on Android and this is a Mac, its a hustle to get the screenshots to the pc. Could use whatsApp but it is currently blocked in my country.
